.babelrc

```diff
@@ -1,9 +1,34 @@
 {
-  "presets": ["es2015"],
-  "plugins": ["transform-decorators-legacy", "transform-class-properties", "transform-runtime"],
+  "presets": [
+    [
+      "@babel/preset-env", 
+      {
+        "targets": {
```

Comment:
Maybe we could do both a modern and a legacy build? I think a good number of people still need to support IE 11.

Reply:
Done. Added es5 (ie11) compatible build